float64 velAY
float64 velBX

